STEPS.md

```
@@ -3639,6 +3640,187 @@ Given the render cache has been cleared
 >  and `@behat-steps-skip:configOverrideBeforeStep`.
 
 
+## Drupal\ConfigTrait
+
+[Source](src/Drupal/ConfigTrait.php), [Example](tests/behat/features/drupal_config.feature)
+
+>  Assert and set stored Drupal configuration values with automatic revert.
```

markdownlint-cli2 (0.22.1)

[warning] Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)